modules/core/src/components/declarative-ui/xviz-plot.js

```diff
@@ -49,6 +50,7 @@ class XVIZPlotComponent extends PureComponent {
     description: PropTypes.string,
     independentVariable: PropTypes.string,
     dependentVariables: PropTypes.arrayOf(PropTypes.string),
+    missingData: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Where is this used?

Reply:
Good catch.. no longer needed - Removing shortly